go.mod

```diff
@@ -31,7 +31,6 @@ require (
 	github.com/golang/mock v1.5.0
```

Comment:
Were these `go.mod` changes committed on purpose?

Reply:
I ran go mod tidy. should I remove the go.mod?

Reply:
It added a `google.golang.org/appengine v1.6.6` dependency which doesn't seem necessary, even with your new code